configuration/persistence.md

```diff
@@ -7,12 +7,238 @@ title: Persistence
 
 # Persistence
 
-Persistence support stores item states over time (a time series).
-openHAB is not restricted to a single data store.
-Different stores can co-exist and be configured independently.
-For further details, please check:
+## Introduction
```

Comment:
Absolutely - otherwise why have the second heading.  I will delete the heading "Introduction".  It is at the beginning - it is introductory text - does not need its own header.

Reply:
Oh - and to answer your initial question, when I was working on the sitemap article I clicked over to persistence and noticed it was basically a stub pointing to the wiki, so I decided to bring the wiki text into this project.  I tried, in the first pass, to correct any glaring issues and to make it OH2 specific, but my main emphasis was just to get something on persistence into the documentation with everything else.

My two cents would be to merge this fairly quickly.  It would be good to have it up and available.  (But it needs to be factually accurate.)